drake/examples/Quadrotor/runDynamics.cpp

```diff
@@ -16,33 +16,60 @@ using namespace Eigen;
 int main(int argc, char* argv[]) {
   const size_t num_lidar_points = 100;
 
-  double final_time = argc >= 2 ? atof(argv[1]) : std::numeric_limits<double>::infinity();
-  cout << "Running simulation for " << final_time << " seconds." << endl;
+  // Get the final time of the simulation
```

Comment:
Missing punctuation (I will abbreviate this as _punc_).
https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar